allure-report-face/src/plugins/report-stats/report-stats-widget/ReportStatsWidget.js

```diff
@@ -0,0 +1,25 @@
+import './styles.css';
+import {ItemView} from 'backbone.marionette';
+import template from './ReportStatsWidget.hbs';
+import ReportWidgetModel from './ReportWidgetModel';
+
+export default class ReportStatsWidget extends ItemView {
+    template = template;
+
+    initialize() {
+        this.model = new ReportWidgetModel();
+        this.model.fetch().then(
+            () => {
+                if (!this.isDestroyed) {
```

Comment:
it should be done with help of `onModel` events. Dunno why i've implement it in such strange way